servlet/src/main/java/io/undertow/servlet/api/DeploymentInfo.java

```diff
@@ -1399,6 +1400,15 @@ public void setPreservePathOnForward(boolean preservePathOnForward) {
         this.preservePathOnForward = preservePathOnForward;
     }
 
+    public int getFileCacheTimeToLive() {
+        return fileCacheTimeToLive;
+    }
+
+    public DeploymentInfo setFileCacheTimeToLive(int fileCacheTimeToLive) {
```

Comment:
Is this here just for the purpose of writing the test case, or is there an intention of making this configurable to the server users?

Reply:
Same explanation than before. It's needed to get the TTL from somewhere because I considered using `CachingResourceManager` was not safe.